Lredir 02

andrewbird opened the pull request:
Update for lredir path permission checks

stsp commented:
Could you please test with /tmp instead
of /bin? Redirecting to /bin looks very
fishy, and in fact, may be broken later
because of a permission checks.
Would be nice if your test makes sure
that lredir to /bin actually fails, so the
checks work.

andrewbird commented:
So I changed the paths to use `/tmp` instead, and added a second test designed to check permission denied if `/tmp` not in access list. I also removed some sort of auto skip failover 'feature' I had implemented before if MFS was unavailable as we can do without magic.

stsp commented:
Marvelous!
Thanks.
